datadog/resource_datadog_dashboard.go

```diff
@@ -5060,6 +5254,49 @@ func buildDatadogTraceServiceDefinition(terraformDefinition map[string]interface
 	return datadogDefinition
 }
 
+func buildTerraformGeomapDefinition(datadogDefinition datadogV1.GeomapWidgetDefinition, k *utils.ResourceDataKey) map[string]interface{} {
+	terraformDefinition := map[string]interface{}{}
+	// Required params
+	terraformDefinition["request"] = buildTerraformGeomapRequests(&datadogDefinition.Requests, k.Add("request"))
+	k.Remove("request")
+
+	if v, ok := datadogDefinition.GetStyleOk(); ok {
+		style := buildTerraformGeomapRequestStyle(*v)
+		terraformDefinition["style"] = []map[string]interface{}{style}
+	}
+
+	if v, ok := datadogDefinition.GetViewOk(); ok {
+		view := buildTerraformGeomapRequestView(*v)
+		terraformDefinition["view"] = []map[string]interface{}{view}
+	}
+
+	if v, ok := datadogDefinition.GetCustomLinksOk(); ok {
+		terraformDefinition["custom_link"] = buildTerraformWidgetCustomLinks(v)
+	}
+
+	if v, ok := datadogDefinition.GetTitleOk(); ok {
+		terraformDefinition["title"] = *v
+	}
+
+	if v, ok := datadogDefinition.GetTitleSizeOk(); ok {
+		terraformDefinition["title_size"] = *v
+	}
+
+	if v, ok := datadogDefinition.GetTitleAlignOk(); ok {
+		terraformDefinition["title_align"] = *v
+	}
+
+	if v, ok := datadogDefinition.GetTimeOk(); ok {
+		// Set to deprecated field if that's what is used in the config, otherwise, set in the new field
+		if _, ok := k.GetOkWith("time"); ok {
+			terraformDefinition["time"] = buildTerraformWidgetTimeDeprecated(*v)
+		} else {
+			terraformDefinition["live_span"] = v.GetLiveSpan()
+		}
```

Comment:
Same here

Reply:
Done in commit [82940d4](https://github.com/DataDog/terraform-provider-datadog/pull/954/commits/82940d4ce6e86936c65dd22e85b98ba68bcc21ef).